### ISSUE TITLE
[FR] Support for ExtraBees serums and Gendustry Gene samples and Genetic templates

### ISSUE THREAD

**Issue by iarspider** [OpenMods/OpenPeripheral#155]
The subject has it all: I'd love to get ability to read information (gene name(s)/value(s), also purity/charges for serums) from EB and Gendustry stuff.

**Comment by MagicTrixor** [comment]
Yup, working on automating bees with computer craft, would be nice if I could read the info from  Gene Sample
